apps/examples/vite.config.ts

```diff
@@ -77,7 +77,7 @@ export default defineConfig(({ mode }) => ({
 		'process.env.TLDRAW_IMAGE_URL': urlOrLocalFallback(
 			mode,
 			env === 'development' ? undefined : 'https://images.tldraw.xyz',
-			8989
+			8786
```

Comment:
why the change here?

Reply:
it was wrong, the images weren't working in the examples app